### ISSUE TITLE
Using JsonContent for a query parameter produces slightly incorrect result

### ISSUE THREAD

**Issue by ikari7789** [zircote/swagger-php#801]
When defining a complex data type in a query parameter, content and the media type should be used, but the `mediaType: <mediaType>` should not be defined.

You can find the example about half-way down this page under "schema vs content."

https://swagger.io/docs/specification/describing-parameters/#schema-vs-content

In their example, they display it as such:

```yaml
    parameters:
      - in: query
        name: filter

        # Wrap 'schema' into 'content.<media-type>'
        content:
          application/json:  # <---- media type indicates how to serialize / deserialize the parameter content
            schema:
              type: object
              properties:
                type:
                  type: string
                color:
                  type: string
```

When converting this into PHPDoc:

```php
/**
 * @OA\Parameter(
 *     in="query",
 *     name="filter",
 *     @OA\JsonContent(
 *         type="object",
 *         @OA\Property(property="type", type="string"),
 *         @OA\Property(property="color", type="string"),
 *     ),
 * )
 */
```

The following is produced:

```yaml
    parameters:
      - name: filter
        in: query
        content:
          application/json:
            mediaType: application/json
            schema:
              properties:
                type:
                  type: string
                color:
                  type: string
              type: object
```

Which is adding `mediaType: application/json` line.

**Comment by DerManoMann** [comment]
Thanks, I've started a new PR to address OAS3 spec conformity which will (eventually) fix this.

https://github.com/zircote/swagger-php/pull/803

**Comment by DerManoMann** [comment]
This was fixed in #822
